build/ci/golangci.yml

```diff
@@ -144,6 +150,7 @@ linters:
     #- gofmt # replaced with gofumpt
     #- nolintlint # strange behavior
     #- gomoddirectives # allow replace directive in go.mod
+    #- misspell - broken, rewrites code
```

Comment:
Broken, renames variables, rewriting cannot be disabled.

Reply:
This is quite usefull plugin which prevents misspelling. I understand that rewriting is annoying however I would rather adopt than drop. 
@jachym-tousek-keboola @hosekpeter what do you think?

Reply:
It is not annoying, it is not working, illogically it overwrites valid names (after the update).

![image](https://github.com/keboola/keboola-as-code/assets/19371734/e759209f-83ce-440d-abe8-4b893c418bbb)

Reply:
I understand, what about using `corrections` file/ `-i` to prevent update `Statuser`.
I think we have some common dictionary that is not part of the linter and it expects that it is a mistake. I can easily create a Jira task to go trough new `misspell` corrections, ignore them and add them to our `Keboola` dictionary.
It seems common issue also for others not just us :)

Reply:
If it overwrites to nonsensical names, I certainly wouldn't want it there because I don't want to have to go through it again to see if something was renamed wrong or something.

Reply:
@Matovidlo please create a task to fix it